src/utils/url.ts

```diff
@@ -1,20 +1,44 @@
 import { DwnError, DwnErrorCode } from '../core/dwn-error.js';
 
-
-export function validateProtocolUriNormalized(protocol: string): void {
+export function validateProtocolUriNormalized(uri: string): void {
   let normalized: string | undefined;
   try {
-    normalized = normalizeProtocolUri(protocol);
+    normalized = normalizeProtocolUri(uri);
   } catch {
     normalized = undefined;
   }
 
-  if (protocol !== normalized) {
-    throw new DwnError(DwnErrorCode.UrlProtocolNotNormalized, 'Protocol URI must be normalized.');
+  if (uri !== normalized) {
+    throw new DwnError(DwnErrorCode.UrlProtocolNotNormalized, `Protocol URI ${uri} must be normalized.`);
   }
 }
 
 export function normalizeProtocolUri(url: string): string {
+  // Keeping protocol normalization as a separate function in case
+  // protocol and schema normalization diverge in the future
+  return normalizeUri(url);
```

Comment:
Non-blocking: what's the pattern for when to use "URI" and "URL" (if there is any). If no specific reason, consider using URL throughout the code for consistency.